__test__/components/__snapshots__/indexPage.test.js.snap

```diff
@@ -25,7 +25,7 @@ Array [
             There is no limit to what you can achieve with a pen.. Write, the world is ready to read.
           </p>
           <button
-            className="index-button sc-htpNat dokJoo"
+            className="index-button sc-htpNat hhMDti"
```

Comment:
I have an issue with this class name that keeps changing. Let's look into it later please.

Reply:
Noted